NitroxModel/MultiplayerSession/PlayerContext.cs

```diff
@@ -14,8 +14,9 @@ public class PlayerContext
     public PlayerSettings PlayerSettings { get; }
     public bool IsMuted { get; set; }
     public NitroxGameMode GameMode { get; set; }
+    public NitroxId DrivingVehicle { get; set; }
```

Comment:
Adding the nullable "?" gives me a warning that the code should be surrounded by #nullable enable and #nullable disable, which I find too big for such an information. Instead I'd go with an xml comment explaining it can be null